Mathlib/GroupTheory/Divisible.lean

```diff
@@ -223,14 +223,12 @@ def rootableByNatOfRootableByInt [RootableBy A ℤ] : RootableBy A ℕ where
   root a n := RootableBy.root a (n : ℤ)
   root_zero a := RootableBy.root_zero a
   root_cancel {n} a hn := by
-    -- Porting note: replaced `norm_num`
     simpa only [zpow_natCast] using RootableBy.root_cancel a (show (n : ℤ) ≠ 0 from mod_cast hn)
```

Comment:
Does norm_num still not work here? If not, isn't this still a valid porting note (perhaps can be linked to the tracking issue of things missing from norm_num)

Reply:
Oops, restored.